src/test/java/com/qdesrame/openapi/test/MaxPageItemsDiffTest.java

```diff
@@ -0,0 +1,38 @@
+package com.qdesrame.openapi.test;
+
+import org.junit.jupiter.api.Test;
+
+import static com.qdesrame.openapi.test.TestUtils.*;
+
+public class MaxPageItemsDiffTest {
+
+    private final String OPENAPI_DOC1 = "max_page_items_diff_1.yaml";
+    private final String OPENAPI_DOC2 = "max_page_items_diff_2.yaml";
+    private final String OPENAPI_DOC3 = "max_page_items_diff_3.yaml";
+
+    @Test
+    void testEqual() {
+        assertOpenApiAreEquals(OPENAPI_DOC1, OPENAPI_DOC1);
+    }
+
+    @Test
+    void testIncreasedMax() {
+        assertOpenApiBackwardCompatible(OPENAPI_DOC1, OPENAPI_DOC2, true);
+    }
+
+    @Test
+    void testDecreasedMax() {
+        assertOpenApiBackwardIncompatible(OPENAPI_DOC2, OPENAPI_DOC1);
+    }
+
+    @Test
+    void testRemovedMax() {
+        assertOpenApiBackwardIncompatible(OPENAPI_DOC2, OPENAPI_DOC3);
```

Comment:
wouldn't removing the max be compatible? its the same as increasing max to infinity

Reply:
Didn't we talk about how some clients could rely on the max to return a certain amount of elements? I don't remember what came out of that discussion

Reply:
that may be true for page_size but i don't think its true for x-max-page-items

Reply:
Ah ok